apps/allocations/app/components/Panel/NewBudget.js

```diff
@@ -139,14 +146,33 @@ class NewBudget extends React.Component {
                 onChange={this.handleSelectToken}
               />
             )}
-
+            <Text
+              css={`
+                white-space: nowrap;
+                line-height: 40px;
+                padding-left: 16px;
+                font-size: 16px;
+                color: ${this.props.theme.contentSecondary};
+              `}
+            >
+              every {timeString(this.props.duration)}
+            </Text>
           </InputGroup>
         </Field>
       </Form>
     )
   }
 }
 
+const timeString = ms => {
+  const days = Math.round(ms/86400000)
```

Comment:
I'd prefer we expand this large number into its constituent pieces, too – we could set up a constant, if you want. `ms / MS_PER_DAY`, where `const MS_PER_DAY = 24 * 3600 * 1000` is set at the top of the file